docs/user-guide/a2-03-wgsl-target-specific.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: user-guide
+permalink: /user-guide/wgsl-target-specific
+---
+
+WGSL specific functionalities
+=============================
+
+This chapter provides information for WGSL (WebGPU Shading Language) -specific functionalities and behaviors.
+
+
+System-Value semantics
+----------------------
+
+The system-value semantics are translated to the following SPIR-V code.
+
+| SV semantic name | WGSL code |
+|--|--|
+| SV_Barycentrics | *Not supported* |
+| SV_ClipDistance<N> | `@builtin(clip_distances)` |
+| SV_CullDistance<N> | *Not supported* |
+| SV_Coverage | `@builtin(sample_mask)` |
+| SV_CullPrimitive | *Not supported* |
+| SV_Depth | `@builtin(frag_depth)` |
+| SV_DepthGreaterEqual | *Not supported* |
+| SV_DepthLessEqual | *Not supported* |
+| SV_DispatchThreadID | `@builtin(global_invocation_id)` |
+| SV_DomainLocation | *Not supported* |
+| SV_GSInstanceID | *Not supported* |
+| SV_GroupID | `@builtin(workgroup_id)` |
+| SV_GroupIndex | `@builtin(local_invocation_index)` |
+| SV_GroupThreadID | `@builtin(local_invocation_id)` |
+| SV_InnerCoverage | *Not supported* |
+| SV_InsideTessFactor | *Not supported* |
+| SV_InstanceID | `@builtin(instance_index)` |
+| SV_IntersectionAttributes | *Not supported* |
+| SV_IsFrontFace | `@builtin(front_facing)` |
+| SV_OutputControlPointID | *Not supported* |
+| SV_PointSize<sup>note</sup> | *Not supported* |
+| SV_Position | `@builtin(position)` |
+| SV_PrimitiveID | *Not supported* |
+| SV_RenderTargetArrayIndex | *Not supported* |
+| SV_SampleIndex | `@builtin(sample_index)` |
+| SV_ShadingRate | *Not supported* |
+| SV_StartVertexLocation | *Not supported* |
+| SV_StartInstanceLocation | *Not suported* |
+| SV_StencilRef | *Not supported* |
+| SV_Target<N> | *Not supported* |
+| SV_TessFactor | *Not supported* |
+| SV_VertexID | `@builtin(vertex_index)` |
+| SV_ViewID | *Not supported* |
+| SV_ViewportArrayIndex | *Not supported* |
+
+*Note* that `SV_PointSize` is a unique keyword that HLSL doesn't have.
+
+
+Behavior of `discard`
+---------------------
+
+Slang's `discard` is translated to `discard` in WGSL, which behaves similar to `discard` in HLSL.
+
+
+Supported HLSL features when targeting WGSL
+-------------------------------------------
+
+Slang supports the following HLSL feature sets when targeting WGSL.
```

Comment:
It says supporting the following, but then the table has many rows that are no. Should reword the text.